packages/suite/src/hooks/wallet/__tests__/useSendForm.test.tsx

```diff
@@ -144,6 +144,26 @@ const actionCallback = (
                     }),
                 }),
             );
+
+            // Previous condition with arrayContaining is checking only partial match:
+            // https://jestjs.io/docs/expect#expectarraycontainingarray
+            // So this is here to check if it was called with any other utxo.
+            [...Object.values(fixtures.UTXO)]
```

Comment:
I know it doesn't look very nice but I can't find any better solution and don't want to spent more time on this test...

Reply:
there is another way how to pick params from recent all of mocked function:
```
const calls = TrezorConnect.composeTransaction.mock.calls.length;
const params = TrezorConnect.composeTransaction.mock.calls[calls - 1][0];
```
and then you can:
```
expect(params.account.utxo.lenght).toEqual(result.composeTransactionParams.account.utxo.length)
```
and do more checks using toMatchObejct instead of objectContaining/arrayContaining

Reply:
thanks, that is definitely easier to read